packages/synapse-react-client/src/components/IconSvg/IconSvg.tsx

```diff
@@ -243,260 +247,263 @@ export type IconSvgProps = {
   icon: IconName
   // If provided, will be shown in tooltip
   label?: string
-  sx?: SxProps
   wrap?: boolean
-}
+} & SvgIconProps
 
-function IconMapping(props: { icon: string; sx?: SxProps }) {
-  const { icon } = props
+function IconMapping(props: { icon: string } & SvgIconProps) {
+  const { icon, ...otherProps } = props
   const color = undefined
 
-  const sx: SxProps = {
+  otherProps.sx = {
     verticalAlign: 'middle',
-    ...props.sx,
+    ...otherProps.sx,
   }
```

Comment:
We are automatically adding `verticalAlign: 'middle'` to `sx`. Ideally, I'd rather get rid of this, but I don't want to check every single icon to see if that breaks something right now, so we leaving it in.